crypto/aes/aes_ige.c

```diff
@@ -38,15 +39,31 @@ void AES_ige_encrypt(const unsigned char *in, unsigned char *out,
                      size_t length, const AES_KEY *key,
                      unsigned char *ivec, const int enc)
 {
+    /* Ignore the return value since we are void */
+    int ret =AES_ige_encrypt_ex(in, out, length, key, ivec, enc);
+
+    assert(ret);
+}
+
+int AES_ige_encrypt_ex(const unsigned char *in, unsigned char *out,
+                       size_t length, const AES_KEY *key,
+                       unsigned char *ivec, const int enc)
+{
     size_t n;
     size_t len = length;
 
+    /* Nothing to do */
     if (length == 0)
-        return;
+        return 1;
 
-    OPENSSL_assert(in && out && key && ivec);
-    OPENSSL_assert((AES_ENCRYPT == enc) || (AES_DECRYPT == enc));
-    OPENSSL_assert((length % AES_BLOCK_SIZE) == 0);
+    if (in == NULL || out == NULL || key == NULL || ivec == NULL)
+        return 0;
```

Comment:
We did not agree on anything other than to agree to disagree :-) ... and the coding style remains silent on the topic ... I.e. we did not encode your view point as the rule. Code that checks isn't rejected and code that doesn't isn't rejected. It is up to each reviewer as to how they respond.